packages/formation/sass/modules/_m-loading-indicator.scss

```diff
@@ -2,10 +2,22 @@
   text-align: center;
 
   .loading-indicator {
-    min-height: 8em;
-    background: url('#{$formation-image-path}/loading-state.svg') center center no-repeat;
-    background-size: contain;
-    text-align: center;
+    display: block;
+    height: 8em;
+    margin-top: 2em;
+
+    &:after {
+      content: " ";
+      display: inline-block;
+      width: 7em;
```

Comment:
Using `em` units, the size of the loader could change if the font-size is set on the container.  Can we look at using the spacing units `units` function for height & width (works here since this isn't a super huge thing), margin, and padding? https://design.va.gov/design/spacing-units